sample/package.json

```diff
@@ -7,7 +7,7 @@
 		"url": "https://github.com/dotnet/vscode-dotnet-runtime.git"
 	},
 	"license": "MIT",
-	"version": "0.0.1",
+	"version": "0.0.9",
```

Comment:
(This is an internal testing extension only so the version doesn't matter much. We could try to keep it concurrent with the extension version.)